libs/fetcher.js

```diff
@@ -28,6 +28,25 @@ function parseParamValues (params) {
     }, {});
 }
 
+function getErrorResponse(err) {
+    var statusCode = err.statusCode || 400;
+    var output = {
+        message: 'request failed'
+    };
+
+    if (typeof err.output !== 'undefined') {
+        output = err.output;
+    } else if (err.message) {
```

Comment:
is this for backwards compatibility? could we just remove the `else` and make the `if (err.message)` its own block?